src/main/java/it/gov/pagopa/cgn/portal/service/ProfileService.java

```diff
@@ -32,6 +32,7 @@ public ProfileEntity createProfile(ProfileEntity profileEntity, String agreement
             throw new InvalidRequestException("A registry already exist for the agreement: " + agreementId);
         }
         profileEntity.setAgreement(agreement);
+        profileEntity.setEntityType(agreement.getEntityType());
```

Comment:
what if we update the profile sending the value with the DTO?

Reply:
profile Entity has no EntityTYpe